docker/demo/config/bench/complex-source.avsc

```diff
@@ -0,0 +1,449 @@
+{
+    "name": "COMPLEX",
+    "fields": [
```

Comment:
I think we can add few more primitive types here i.e replace some fields to bytes/int/boolean for coverage?https://avro.apache.org/docs/1.8.2/spec.html#schema_primitive 
Does this have complex cases like Maps or Arrays of Maps or Arrays of structs . can we change some of these fields to cover these? This will give us coverage across issues.